datasetinsights/datasets/__init__.py

```diff
@@ -3,7 +3,7 @@
 from .coco import CocoDetection
 from .groceries_real import GoogleGroceriesReal, GroceriesReal
 from .nyudepth import NyuDepth
-from .synthetic import SynDetection2D
+from .synthetic import SynDetection2D, read_bounding_box_2d
```

Comment:
I think `read_bounding_box_2d` method should be in `io/bbox.py`, cause it's related to 2d bounding box.

Reply:
I think there are arguments that can go either way. This method reads bounding box from synthetic bounding box format to `BBox2D`. So it's related to 2D object detection as well. I don't feel strong one way or the other, as long as the module is well documented.

Reply:
Okay. Sounds good.

Reply:
Synthetic is using the bbox to transform into bbox representation so I think its right place to keep.